[Finish] Add support for simplified chinese 简体中文化

dtlnor opened the pull request:
same as pr #3018, I mess-up the git history so I close that one and re-open a pr again.
I've finished all the translation up to f49c08ea566385db339c6628f65c3a121033f67c, welcome for any improvement

dtlnor committed:
Create zh-hans.json

ruocaled commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,456 @@
+    "Save as float16": "以 float16 储存",
+    "See": "查看",
+    "wiki": "wiki",
+    "for detailed explanation.": "想看详细解释的话",
```
![image](https://user-images.githubusercontent.com/873853/197295734-afd83317-64c3-4f91-b2fe-024f1cf05481.png)

These 3 parts are used together in one sentence in UI

“查看wiki想看详细解释的话” feels a bit weird, maybe "查看wiki以了解详细说明" ?

dtlnor replied:
thank you, didn't notice this when working with json file. I'll correct this.

CCRcmcpe commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,456 @@
+    "view": "查看",
+    "api": "api",
+    "•": "•",
+    "built with gradio": "基于 Gradio 库编译",
```
基于 Gradio 构建

dtlnor replied:
get it, didn't notice that Gradio isn't a library but a build tool

CCRcmcpe commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,456 @@
+    "Modules": "模组",
+    "Enter hypernetwork layer structure": "输入 hypernetwork 层结构",
+    "Add layer normalization": "添加层标准化",
+    "Overwrite Old Hypernetwork": "Overwrite Old Hypernetwork",
```
覆写旧的 Hypernetwork

CCRcmcpe commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,456 @@
+    "Max steps": "最大迭代步数",
+    "Save an image to log directory every N steps, 0 to disable": "每 N 步保存一个图像到日志目录，0 表示禁用",
+    "Save a copy of embedding to log directory every N steps, 0 to disable": "每 N 步将 embedding 的副本保存到日志目录，0 表示禁用",
+    "Save images with embedding in PNG chunks": "保存图像並在 PNG 文件中嵌入 embedding 文件",
```
“并”

CCRcmcpe commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,456 @@
+    "What to put inside the masked area before processing it with Stable Diffusion.": "在使用 Stable Diffusion 处理蒙版区域之前要在蒙版区域内放置什么",
+    "fill it with colors of the image": "用图像的颜色填充它",
+    "keep whatever was there originally": "保留原来的东西",
+    "fill it with latent space noise": "用 latent 空间的噪声填充它",
```
latent空间->潜空间

dtlnor replied:
~~a bit weird to me to call latent space as 潜空间, I've struggling about these terms very much~~ looks like 潜空间 is a convention translation to latent space, I'll coorect it. Thankyou.

CCRcmcpe commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,456 @@
+    "Preprocess images": "图像预处理",
+    "Name": "名称",
+    "Initialization text": "初始化文字",
+    "Number of vectors per token": "每个令牌(token)的向量数",
```
“令牌”不达意，不如直接删除用“token”

dtlnor replied:
token確實是太難翻譯了，~~我本來想符記可能像一點，不過看到隔壁ai繪畫wiki book用令牌我就躺平了~~，果然還是直接用原詞算了

CCRcmcpe commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,456 @@
+    "Train": "训练",
+    "Settings": "设置",
+    "Prompt": "提示词",
+    "Negative prompt": "否定提示词",
```
负面提示词

dtlnor replied:
你认为反向提示词如何呢？因为我看不少人用咒语、反咒描述这个terms。负面提示词感觉像是只是让人填写bloody、upset的那种感觉。所以我直接取了negative表示否定的意思进去。

CCRcmcpe replied:
那就”反向提示词“吧

CCRcmcpe commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,456 @@
+    "Overwrite Old Embedding": "覆写旧的 Embedding",
+    "Source directory": "源目录",
+    "Create images embedding": "生成 embedding 图片",
+    "Modules": "模组",
```
模块

CCRcmcpe commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,456 @@
+    "Width": "宽度",
+    "Height": "高度",
+    "Restore faces": "面部修复",
+    "Tiling": "密铺",
```
tile->块，tiling->分块

dtlnor replied:
https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/3018#issuecomment-1281884089
在这里有人提到过，分块可能更直观，密铺可能会更严谨？

CCRcmcpe replied:
密铺是会看起来更严谨但人第一眼看上去不知所谓，不如英文原文拍上去吧

dtlnor replied:
我改成图块以及分块了，同时在后加加上了英文注译。~~只能说tiling这个东西，懂得都懂吧~~

CCRcmcpe commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,456 @@
+    "Prompt": "提示词",
+    "Negative prompt": "否定提示词",
+    "Run": "运行",
+    "Skip": "略过",
```
跳过

dtlnor replied:
(同义词我就不修改了)

CCRcmcpe replied:
然而跳过是这种操作的惯用翻译

dtlnor replied:
明白了, 最新的commit使用了这个翻译

CCRcmcpe commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,456 @@
+    "Denoising strength change factor": "降噪强度的调整因数",
+    "Recommended settings: Sampling Steps: 80-100, Sampler: Euler a, Denoising strength: 0.8": "推荐设置：采样迭代步数：80-100，采样器：Euler a，降噪强度：0.8",
+    "Pixels to expand": "拓展的像素",
+    "Outpainting direction": "外填充的方向",
```
怎么一下翻译成外补绘制一下翻译成外填充

dtlnor replied:
mistake，之前用的外填充，后来发现外补绘制是更合适的翻译

CCRcmcpe commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,456 @@
+    "Seed of a different picture to be mixed into the generation.": "将要参与生成的另一张图的种子",
+    "How strong of a variation to produce. At 0, there will be no effect. At 1, you will get the complete picture with variation seed (except for ancestral samplers, where you will just get something).": "想要产生多强烈的变化。 设为 0 时，将没有效果。 设为 1 时，你将获得完全产自偏差种子的图片（ancestral 采样器除外，你只是生成了一些东西）",
+    "Make an attempt to produce a picture similar to what would have been produced with same seed at specified resolution": "尝试生成与在指定分辨率下使用相同种子生成的图片相似的图片",
+    "This text is used to rotate the feature space of the imgs embs": "此文本用于旋转imgs embs的特征空间",
```
imgs embs=图片embeddings

Yorha4D commented:
hi~我建议，各个翻译者们不要继续内耗了，定个时间可以开一个线上会议，讨论下，目前这个版本里很多词难以统一，比如upscale超采样 upscaler超采样器这些在行业里已经用了很多年了，还有一些功能理解上导致翻译错误..应该咱们去讨论下
![image](https://user-images.githubusercontent.com/116185401/197336646-8ae12f1d-48d5-4166-b908-4b81035c4ff4.png)
 
从明天20点开始，准时等人啦
只剩一瓶辣椒酱 邀请您参加腾讯会议
会议主题：只剩一瓶辣椒酱预定的会议
会议时间：2022/10/23 20:00-20:30 (GMT+08:00) 中国标准时间 - 北京
重复周期：2022/10/23-2022/10/29 20:00-20:30, 每天

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/dm/QjqsFOIR9xJl

#腾讯会议：628-5563-1297

dtlnor committed:
Update zh-hans.json

dtlnor commented:
> @dtlnor hi~我建议，统一下翻译，让各个翻译者们不要继续内耗了(有时间可以做点别找找BUG啥的，现在有一些用了翻译换了词就找不到函数了)，定个时间可以开一个线上会议，讨论下，目前这个版本里很多词难以统一，比如upscale超采样 upscaler超采样器这些在行业里已经用了很多年了，还有一些功能理解上导致翻译错误..应该咱们去讨论下 ![image](https://user-images.githubusercontent.com/116185401/197336646-8ae12f1d-48d5-4166-b908-4b81035c4ff4.png)
> 
> 从明天20点开始，准时等人啦 只剩一瓶辣椒酱 邀请您参加腾讯会议 会议主题：只剩一瓶辣椒酱预定的会议 会议时间：2022/10/23 20:00-20:30 (GMT+08:00) 中国标准时间 - 北京 重复周期：2022/10/23-2022/10/29 20:00-20:30, 每天
> 
> 点击链接入会，或添加至会议列表： https://meeting.tencent.com/dm/QjqsFOIR9xJl
> 
> #腾讯会议：628-5563-1297

upscale/upconverting/upsampling (even supersampling) on image processing as a "tech term" are mostly references to the same action. Same as in Chinese, 放大、上轉換、上採樣、升評、超採樣, they are just an alternative to describe the process of enlarging an image digitally.

dtlnor committed:
Update zh-hans.json

- fix translation of Tile / Tiling
- fix translation of Negative prompt

Yorha4D commented:
> > @dtlnor嗨~
> 
> 图像处理中的升级/上转换/上采样（甚至是超级采样）作为“技术术语”，主要是指相同的操作。与中文中一样，放大、上轉換、上採樣、升頻、超採樣，它们只是描述以数字方式放大图像的过程的替代方案。

但目前的翻译里对于upscaler和upscale并没有做到翻译的统一，同时放大这个词汇太过于宽广，推荐再参考一下常用CG软件翻译斟酌一下，此外当前翻译造了很多新词，这是在过于max、maya或者blender中未曾使用过的，但它们表述的是一个意思

Yorha4D commented:
@dtlnor 目前感觉最明显的问题在于Seed，在当前的版本中被翻译成种子、随机种子和随机种，这应该做到统一并建议使用C4D\BLENDER\MAYA里的惯用词随机种，这样会使用户更加便于理解，同时避免切换模块导致不一致带来歧义(一定会有这样的使用者)，还有一些其他问题，希望能明天晚上20点，在上面的腾讯会议里，沟通一下~
这是我们公司的动画-https://www.bilibili.com/bangumi/play/ep317088
如果有必要下周可以喊一些技术导演一起讨论下，避免之后出现反复修改

toppac commented:
看了这些翻译我觉得还是用辣椒酱的比较亲民。。翻译的太学术性了

dtlnor commented:
> 

I agree with your words, we want to make the community better. But I am sorry that I have no interest to have a meeting with you or anyone in your company. I can't manage to understand the need to meet with your director. Your role or any others' role does not affect the accuracy of the translation. Unless that person is a certificated translator. Let the discussion focus on the translation itself. Let every word and every sentence can be recorded here, thus everyone is able to review our conversation.

dtlnor commented:
> @dtlnor 目前感觉最明显的问题在于Seed，在当前的版本中被翻译成种子、随机种子和随机种，这应该做到统一并建议使用C4D\BLENDER\MAYA里的惯用词随机种，这样会使用户更加便于理解，同时避免切换模块导致不一致带来歧义(一定会有这样的使用者)，还有一些其他问题，希望能明天晚上20点，在上面的腾讯会议里，沟通一下~ 这是我们公司的动画-https://www.bilibili.com/bangumi/play/ep317088 如果有必要下周可以喊一些技术导演一起讨论下，避免之后出现反复修改 千言万语 让我们一起把它打磨的更好吧~



> > > @dtlnor嗨~
> > 
> > 
> > 图像处理中的升级/上转换/上采样（甚至是超级采样）作为“技术术语”，主要是指相同的操作。与中文中一样，放大、上轉換、上採樣、升頻、超採樣，它们只是描述以数字方式放大图像的过程的替代方案。
> 
> 但目前的翻译里对于upscaler和upscale并没有做到翻译的统一，同时放大这个词汇太过于宽广，推荐再参考一下常用CG软件翻译斟酌一下，此外当前翻译造了很多新词，这是在过于max、maya或者blender中未曾使用过的，但它们表述的是一个意思

Thank you for your advice, this is my mistake, I'll correct it and commit.

dtlnor committed:
Update zh-hans.json

- unify the translation of upscale / upscaler
- unify the translation of seed / random seed
- correct images embedding's translation

dtlnor commented:
> 看了这些翻译我觉得还是用辣椒酱的比较亲民。。翻译的太学术性了

ummm 有没有什么建议，我看看有没有哪些可以替代为更简单易懂的翻译

Yorha4D commented:
@dtlnor 啊这，我的表述可能有些过于正式和强硬了(一打字就容易这样...略微致命)，先让我用这个图表述下我此时的心情
![image](https://user-images.githubusercontent.com/116185401/197357898-7ecca42e-d265-4a66-ab5d-587cb8be8a33.png)

我重新描述下我的本意：
1对于当前翻译9成以上是认同的，而且发现有一些接地气的词条比如roll翻译成抽，这样大家duang的一下就能理解了，很NICE。
2但美中不足的地方，首先是一致性问题，我刚看了下新递交的内容发现有所改善，看来这个我相信是你的话，自己检索就应该可以搞定
3然后就是一些词，作为3D从业者感觉到一些陌生，一些用了很多年的词汇被翻译成新词汇，虽然知道是什么功能，但总想挣扎一下（比如平铺这个用的真的是出感情了ORZ..）........
4最后就是有几处，相对于功能，在翻译之后他可能让使用者迷茫，这也是我想和你通话的第一个原因，沟通一下下..说不定大家就都豁然开朗了

然后就是接下来推进这个项目的中文翻译，很明显不需要如此之多的中文分支，而且项目管理者也表示只能合并一个。
所以把我那个关了或者迁到这里都是可以的，但我还是想为这个翻译做一些工作的.这是想沟通的第二个原因
因为有一些时候，大家的思路碰碰电波才能擦出一些奇妙的火花

希望能有这个机会

23号20点.-》腾讯会议碰碰电波啦（有可能的话也可以聊聊别的开源哈哈）：
https://meeting.tencent.com/dm/QjqsFOIR9xJl
![image](https://user-images.githubusercontent.com/116185401/197358512-ebc3b8fd-6570-46c6-97d3-e7fb10e8ad51.png)

Yorha4D commented:
@dtlnor 给我点时间我把那些感觉有问题的地方整理下，尽快给到你，或许有一些帮助

dtlnor committed:
Update zh-hans.json

- update to latest

Yorha4D commented:
————————————————————————————————————————————————————
样式不统一

seed-当前-存在随机种、随机种子、种子三种翻译，在很多长句翻译里面都翻译成了种子，建议统一为字数最少的`  随机种
Crop and resize和Resize and fill-目前翻译成了 裁剪后缩放 和 缩放并填空-建议押韵统一，统一翻译成 XX并XX(and一般没先后的意思)
image/img-存在图像和图片两种，建议改为图像（只需要改两处）
Restore faces-存在面部修复和面部恢复、人脸修复、人脸恢复好几种，建议统一为面部修复，因为这个词当时看了下以来的项目，他们更倾向于repairXX
noise/denoising-存在噪波和噪声，建议同意为噪波，噪声这个指的是一种声学现象，而不是数学概念
skip-目前存在跳过和略过，建议统一化
fill-目前有填充和填空两种，我看到当前更倾向于填空，但建议使用填充-这个词在PS\AE\画图\以及各类软件中和它当前在webui的表现来看都是填充...（要知道填空这个词...上一次见它还是在答填空题和的时候，某种意义上填充倾向于填满，填空更倾向于空）
![image](https://user-images.githubusercontent.com/116185401/197361075-e35369d5-e28a-4e44-8832-b60c603b9c60.png)
![image](https://user-images.githubusercontent.com/116185401/197361029-faf60286-ba09-4637-a6fe-dd64d3067e41.png)
latent -目前存在潜空间和没翻译的情况
Interrogate -涉及到这个词有反求、推断两种描述，建议统一化，同时这个词更多的是对对象的询问、查询的意思，反求和推断是另外一些东西

——————————————————————————————————————————————————
描述不准确

checkpoint和model还是有一些区别的，下文中也有长句将二者分开
Extras-当前为后处理，此处对应的应该是 Extra 面板，面板目前有超采样和人脸修复功能，这些更加相当于是附加的功能而不是after effect建议保留原本的 附加 含义以方便后期拓展
Image Browser-图像浏览器  图库是另外一个词和定义
Interrupt-这个应该是终止，关于目前的中断是指计算机运行过程中，出现某些意外情况需主机干预时，机器能自动停止正在运行的程序并转入处理新情况的程序，处理完毕后又返回原被暂停的程序继续运行(百度百科)，然后SDW这个功能并不能恢复到点击之前，所以不是中断
tiling/tile/tiles/tiled-这一族词，tiling建议还是平铺....毕竟window桌面壁纸它...一直是平铺、后来延续到各个软件（惯用词）
然后tiles这个才是一片一片组起来的基本单元分块，然后tile当动词就是各种铺的姿势了，比如平铺、铺地砖，然后tiled是动词的过去式
Firstpass height/width-第一遍高度-高分辨率修复，主要是应对大分辨率成像容易导致的构图分散，所以他会先生成1个低分辨率的图像，再以此为基础进行提升细节，在这里指的是先创建的低分辨率图像的尺寸，建议-预处理或第一步高度/宽度
Denoising strength-Denoising这个属于惯用词了-降噪，英伟达最早开的锅
Variation相关词语（Variation seed、Variation strength、Var. seed、Var. strength）-这个词的本意并不是差异（difference），他主要是变体、变种、变量和变化的意思，而且对于功能它的目的也是在原始随机种基础上加上变量强度，使图像有变化。建议-使用变体
Batch count和Batch size-这两个词在翻译的时候应该有所关联，目前批量数量和批量大小放在一起很模糊，应该像有几个盒子，一个盒子里有几支笔这样清晰明了-建议干脆发挥汉字魅力批就是批，量就是量-- 批次(多少批)  批量（一批多少量）
Aesthetic text for imgs的img并没有图像外的意思，在这里用的还是复数，因此并不能翻译为图类（没有type）....
Draw legend-这个应该不是在图表中包括轴标题，检查下是不是粘贴错了
resize-这个保留意见-常见的是调整尺寸..缩放scale是这个，勉强说得过去
Poor man's outpainting-当前翻译成Poor man 的外补绘制，有点勉强，这个词直译的话是穷人的外补绘画，但poor man是一个比喻的用法，建议翻译成低成本外补绘画
Override一些参数的时候-改为重设某某参数为初始值更合适，而不是覆写
Randomness-当前随机强度可以优化下-随机度
factor-系数
Pixels to expand-当前翻译为拓展的像素-建议严谨一些-拓展的像素数
Original prompt-当前翻译为原始提示词，但一般习惯就和初始值、初始化、初始状态一样，更倾向初始提示词
Scale by和Scale to-这俩实际功能第一个是按比缩放第二个是缩放到指定尺寸，在这里翻译成缩放到和缩放为，其实都是scale to的意思，scale by这个不押韵也得改-等比缩放
Images directory-这里翻译为图像路径是不对的路径值得是path，而这里主要指的是那个文件存储的文件夹
number of images to delete consecutively next-这里用机翻也行-接下来要连续删除的图像数，现在这个下一批要批量删除的图像数太绕了
favorites-windos标准用语-收藏夹，指的是存放收藏物品的地点，而不是收藏这个行为或者里面的藏品
ignore-现在的翻译是无视.....但无视一般是人类才能做到的，建议使用忽略更加得体
Save a copy of image before doing face restoration-在进行面部修复之前保存图像副本
Unload VAE and CLIP from VRAM when training-当前翻译为训练时从显存(VRAM)中取消 VAE 和 CLIP 的加载略微有点绕，直接训练时卸载VAE和CLIP
Filename join string-当前翻译为文件名连接用字符，但string是字符串而不是char字符
When reading generation parameters from text into UI (from PNG info or pasted text), do not change the selected model/checkpoint.这句话当前翻译"当从文本读取生成参数到 UI（从 PNG 信息或粘贴文本）时，不要更改选定的模型(ckpt)，，其中model和checkpoint是两个不同的东西，不要都翻译成模型
Do not do anything special-当前翻译为-并没有做什么特别的事情，但这里是一些通用按钮的提示，例如风格 是来提示用户的意思 无需做特别的事情而不是表述状态

————————————————————————————————————————————
整体上，差不多就是这些了，建议检查一下，同时一些词语，特别是中文不建议造词，和使用生僻词标新立异，应该遇到的时候多看看其他工具的翻译情况，同时一些专有名词不要混淆，特别是他们同时出现的时候比如model和checkpoint是两个不同的东西.......
以上，多检查检查，肯定还有别的需要注意的地方......
期待你的好消息~（通宵码字..可能有错别字见谅）
![image](https://user-images.githubusercontent.com/116185401/197362913-355ac5e9-67d6-48c2-b02b-efc6ad111b08.png)

liggest commented:
翻译这事很难完全统一的吧，尤其是既要亲民又要有专业性，感觉有点不可兼得…
比如 prompt，要是随便一点我觉得叫咒文也不错，但论专业性的话，目前所用的提示词可能都差点意思（毕竟 prompt 里面实际上不只能写以逗号分隔的 tag，还可以写完整句子，叫提示词反而有些误导了），妥协一点叫提示文本，又觉得有点长
类似的例子感觉还有不少，每个人想要的翻译大概不太一样，比如我作为没有3D从业背景的人，就觉得 noise 译成噪波令人困惑

Yorha4D commented:
> 翻译这事很难完全统一的吧，尤其是既要亲民又要有专业性，感觉有点不可兼得… 比如 prompt，要是随便一点我觉得叫咒文也不错，但论专业性的话，目前所用的提示词可能都差点意思（毕竟 prompt 里面实际上不只能写以逗号分隔的 tag，还可以写完整句子，叫提示词反而有些误导了），妥协一点叫提示文本，又觉得有点长 类似的例子感觉还有不少，每个人想要的翻译大概不太一样，比如我作为没有3D从业背景的人，就觉得 noise 译成噪波令人困惑
@liggest 这个翻译是面向所有人的，应该慎重且严谨，对于noise，是一个惯用词,这样大家看到的时候才能更加容易接受
在MAX\BLENDER\MAYA\C4D中noise是统一的，包含了9成以上的玩家.......
![image](https://user-images.githubusercontent.com/116185401/197380421-1dcb5658-46b5-4e5b-8218-fb57b335d7f8.png)
![image](https://user-images.githubusercontent.com/116185401/197380455-19e6159f-6b55-4d70-9337-27b17ea16672.png)
![image](https://user-images.githubusercontent.com/116185401/197380602-7ef1f385-47fb-4144-96f3-7e775c44a716.png)
![image](https://user-images.githubusercontent.com/116185401/197380625-dfb0aa38-79d7-42c4-b294-86ea71285179.png)

liggest commented:
所以我说了，我不是一个 3D 行业的从业者，这对我来说不是惯用词
这是一个与机器学习有关的项目，为何要以 3D 行业的通常译名作为标准？
我不觉得参与讨论的人中使用上述软件的人会有你说的那么多，希望不要用你自己来代表所有人

Yorha4D commented:
> 机器学习 投票发B站了可以来看看最后的结果吧
https://t.bilibili.com/720153878426812418?spm_id_from=444.41.0.0

Yorha4D commented:
> > 机器学习 投票发B站了可以来看看最后的结果吧
 @liggest https://t.bilibili.com/720153878426812418?spm_id_from=444.41.0.0

lockmatrix commented:
@Yorha4D 我个人也不赞成说仿照Autodesk的译法来，这类软件译法有时候奇怪也不少见了。
就算排除各个领域的习惯，noise本身翻译成噪声我觉得没问题。搞个噪波就比较奇怪。

lockmatrix commented:
> 这个翻译是面向所有人的，应该慎重且严谨，对于noise，是一个惯用词,这样大家看到的时候才能更加容易接受
在MAX\BLENDER\MAYA\C4D中noise是统一的，包含了9成以上的玩家.......

个人觉得绝大多数人中，用MAX\BLENDER\MAYA\C4D的占比并不大，应该小于所有学过《信号与系统》或接触过机器学习的人，所以从领域投票来说，我觉得后者更恰当一些。

再考虑到非专业领域的习惯，noise应该是更加慎重且严谨的。

Yorha4D commented:
> 所以我说了，我不是一个 3D 行业的从业者，这对我来说不是惯用词 这是一个与机器学习有关的项目，为何要以 3D 行业的通常译名作为标准？ 我不觉得参与讨论的人中使用上述软件的人会有你说的那么多，希望不要用你自己来代表所有人 大可去百度一下“机器学习 噪波”，看看有什么结果

@liggest 3D、画师这些群体未来会成为主要用户，所以才希望用熟悉的词语，退一万步，搁置一下，翻译中既有噪声又有噪波，最起码应该统一，而不是看起来像机翻......麻了...
![image](https://user-images.githubusercontent.com/116185401/197382173-026a3de1-74cd-4b72-88c3-a5c734c7bf6f.png)

lockmatrix commented:
> 3D、画师这些群体未来会成为主要用户，所以才希望用熟悉的词语

我觉得画师、一般用户、ML相关群体会成为主要用户，3D这个嘛，感觉在支持3D模型之前应该不会是主要的。

lockmatrix commented:
@Yorha4D 另外建议引用的时候区分一下引文和你的回复，现在是混在一起的，看起来容易混淆，也不符合你的严谨的目标。

在引用文行和回复行之间需要有个空行，否则github会认为你整体一段都是引文。

liggest commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,482 @@
+    "Style 2": "模版风格 2",
+    "Label": "标签",
+    "File": "文件",
+    "Drop File Here": "拖拽文件到这里",
```
```json
    "Drop Image Here": "拖拽图像到此",
    "Drop File Here": "拖拽文件到这里",
```
```suggestion
    "Drop File Here": "拖拽文件到此",
```

liggest commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,482 @@
+    "CodeFormer visibility": "CodeFormer 可见度",
+    "CodeFormer weight (0 = maximum effect, 1 = minimum effect)": "CodeFormer 权重 (0 = 最大效果, 1 = 最小效果)",
+    "Open output directory": "打开输出目录",
+    "Send to txt2img": "输出到文转图",
```
```json
    "Send to txt2img": "输出到文转图",
    "Send to img2img": "传送到图转图"
```

```suggestion
    "Send to txt2img": "传送到文转图",
```

Yorha4D commented:
> @Yorha4D 另外建议引用的时候区分一下引文和你的回复，现在是混在一起的，看起来容易混淆，也不符合你的严谨的目标。
> 
> 在引用文行和回复行之间需要有个空行，否则github会认为你整体一段都是引文。

ok，这一点没有注意到。
然后关于噪波/噪声，对于我来说一切都可以接受，希望能够在这个词出现的时候统一一下（语义相同的时候）

Yorha4D commented:
@liggest @lockmatrix @dtlnor 话说...今晚20点来个会议吧..聊聊天讨论讨论，感觉能在此相遇也算是一份机缘了
腾讯会议https://meeting.tencent.com/dm/QjqsFOIR9xJl

lockmatrix commented:
> 看了这些翻译我觉得还是用辣椒酱的比较亲民。。翻译的太学术性了

其实翻译版本可以有多个的，fork一个，或者开一个汉化包都没问题，导航还支持换声音呢。

不过这个目前还算是项目的官方翻译，严谨一点应该是对的。游戏的官方翻译如果整活的话都还是被喷了。
（不过现在是个全民喷的时代，无论怎么样都会有人觉得有问题，相对来说，“古板”一点被喷的最少。）

lockmatrix commented:
> hi~我建议，统一下翻译，让各个翻译者们不要继续内耗了(有时间可以做点别找找BUG啥的，现在有一些用了翻译换了词就找不到函数了)，定个时间可以开一个线上会议，讨论下，目前这个版本里很多词难以统一

> 话说...今晚20点来个会议吧..聊聊天讨论讨论，感觉能在此相遇也算是一份机缘了
腾讯会议https://meeting.tencent.com/dm/QjqsFOIR9xJl

@Yorha4D  唔……github其实自己就是一个异步协同的平台，在这里拉同步的会议怪怪的，估计时间也难以匹配。
真想讨论的话建议开个群聊。github的PR相对还是不适合这种讨论的。

liggest commented:
> > 看了这些翻译我觉得还是用辣椒酱的比较亲民。。翻译的太学术性了
> 
> 其实翻译版本可以有多个的，fork一个，或者开一个汉化包都没问题，导航还支持换声音呢。
> 
> 不过这个目前还算是项目的官方翻译，严谨一点应该是对的。游戏的官方翻译如果整活的话都还是被喷了。 （不过现在是个全民喷的时代，无论怎么样都会有人觉得有问题，相对来说，“古板”一点被喷的最少。）

我也觉得可能分多个版本比较好（官方译名、整活版、简版、面向艺术工作者的版本…），毕竟重口难调
现在感觉这么做比较大的阻碍是，仓库更新太快了，本地化文件不太稳定
- 词条变化的比较频繁，有些刚填完下个版本就移除了，目前这个版本中也有些已经用不上了的词条，不知需不需要移出去
- 有些词条翻译了也不会体现在网页中，不知是不是 bug
- 点网页中有些按钮后再下载本地化模板，可能里面会多出来一些额外内容，每个人经过不同操作得到的本地化模板可能是不一样的

如果能解决其中的一些，可能翻译会更顺畅吧 \_(:з」∠)\_

Yorha4D commented:
@liggest @dtlnor @lockmatrix 我创建了讨论Q群-616878364
![image](https://user-images.githubusercontent.com/116185401/197383586-8814b494-b06c-47ff-b47f-0e8f6cf9918e.png)

liggest commented:
本地化文件中无论是 `grid` 还是翻译后的 `概览图` 我都不太理解是什么意思，想请问和它相关的功能主要在网页的什么地方呀？
我被别人建议翻译为 `宫格图`，感觉更贴近 `grid` 格子的意思，似乎直观一些
但我至今也没找到在哪用和它相关的功能，不知道具体是否贴切

lockmatrix commented:
> 本地化文件中无论是 grid 还是翻译后的 概览图 我都不太理解是什么意思，想请问和它相关的功能主要在网页的什么地方呀？
我被别人建议翻译为 宫格图，感觉更贴近 grid 格子的意思，似乎直观一些
但我至今也没找到在哪用和它相关的功能，不知道具体是否贴切

就是那个产出多个图之后，自动把它们拼成一个网格排列的大图片的功能，方便浏览用的。

liggest commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,482 @@
+    "Show images zoomed in by default in full page image viewer": "在整页图像查看器中默认放大显示图像",
+    "Show generation progress in window title.": "在窗口标题中显示生成进度",
+    "Quicksettings list": "快速设置列表",
+    "Localization (requires restart)": "本地化（需要重新启动）",
```
```suggestion
    "Localization (requires restart)": "本地化（需要重新启动）",
    "ja_JP": "日文",
    "zh-hans": "中文",
```

鉴于日文已合并，可以更好地支持在应用中文本地化后切换语言

dtlnor replied:
这个我个人倾向不做任何翻译。保持文件名就可以了。大多数的用户都懂得自己语言的代码简写。

liggest commented:
> 就是那个产出多个图之后，自动把它们拼成一个网格排列的大图片的功能，方便浏览用的。

原来如此…是我想复杂了，没想到指的是那个，感谢说明

Yorha4D commented:
> 本地化文件中无论是 `grid` 还是翻译后的 `概览图` 我都不太理解是什么意思，想请问和它相关的功能主要在网页的什么地方呀？ 我被别人建议翻译为 `宫格图`，感觉更贴近 `grid` 格子的意思，似乎直观一些 但我至今也没找到在哪用和它相关的功能，不知道具体是否贴切

![image](https://user-images.githubusercontent.com/116185401/197384048-60c03e42-3a83-4c8b-90d6-c3b0262ce87d.png)

liggest commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,482 @@
+    "Send to extras": "传送到后处理",
+    "Make Zip when Save?": "保存时生成压缩档?",
+    "Textbox": "文本框",
+    "Interrogate\nCLIP": "推断提示词\nCLIP",
+    "Interrogate\nDeepBooru": "推断提示词\nDeepBooru",
```
```json
    "Interrogate Options": "逆向生成提示词选项",
    "Interrogate\nCLIP": "推断提示词\nCLIP",
```

```suggestion
    "Interrogate\nCLIP": "CLIP\n逆向生成提示词",
    "Interrogate\nDeepBooru": "DeepBooru\n逆向生成提示词",
```

dtlnor replied:
确实需要统一，唯一的问题是逆向生成太长，按钮会很丑（\n不换行啊啊啊啊）。但后面设定全用推断感觉又不太合适。。。好吧，按钮丑一点就丑一点吧

liggest replied:
想要简化的话，或许可以试试“反推”

dtlnor replied:
> 想要简化的话，或许可以试试“反推”

sweet!

liggest commented:
> ![image](https://user-images.githubusercontent.com/116185401/197384048-60c03e42-3a83-4c8b-90d6-c3b0262ce87d.png)

了解了，谢谢

dtlnor commented:
> 图像

seed - 应该已经在上一个commit fix了。
Crop and resize和Resize and fill - fixed
image/img - 这个确实是我翻译新的句子的时候引入的问题。目前打算只留下一句`“尝试生成与在指定分辨率下使用相同随机种子生成的图像相似的图片”` 只是为了避免重复用同一个词引起的审美疲劳。图片跟图像反正是同一个意思。
Restore faces - 统一为面部修复
noise/denoising - 请提出噪声=声学概念。噪波=数学概念的依据。在我看来这两者用来表达noise这个概念都能理解。但前者更多人使用。特别在图像处理领域，以我所知，噪声比“噪波”的使用频率比较高。
skip - 统一为跳过
fill - fill 的提示是 fill it with colors of the image，用图像的颜色填充它。效果是直接把那个被蒙住的内容抹掉（并填充图片的颜色）然后生成的可能是毫不相干的东西。在这种考虑下我才把名字改成填空，然后在info里面以“用图像的颜色填充它”进行说明
latent - 潜空间和没翻译的情况请问在哪一行，我搜索了一下没有看到
Interrogate - 这个词我选择了以实际功能意译回来，因为原文英文没有很好很贴切的翻译。我全改成推断吧
checkpoint和model还是有一些区别的。这个我非常清楚。所以我直接括号了ckpt。这样的好处是懂的人都懂。但你翻译成检查点或者任何checkpoint的中文翻译都很难让新手理解这是什么东西。本身checkpoint就是model的一种借代
Extras - 目前依旧是后处理为主所以翻译为后处理（即使是附加功能那也是对diffusion之后的结果进行的处理）。如果后面拓展了extras的功能再改翻译为附加功能就是了。后处理以目前的功能来说更加一目了然
Image Browser - 因为他并不只是浏览图片用的，他并不是一个给你看图片的“图片浏览器”。他是让你“浏览你生成了哪些图片”的。他会直接读你的文件夹里面的图。以功能翻译的话图库更合适。（在更新之前，他是直接整个文件夹的图都一次性载入的，现在分页了但我没看代码是不是还是整个图库都载入了）
Interrupt - 你说的没错，考虑到运行到一半立即停止的含义。翻译为中止。
tiling/tile/tiles/tiled - 在3d软件或者windows里面自然可以很容易理解平铺是用来干嘛的，把图片平铺到任意大小的画布（可以是背景）中。但这里的tiling跟前述的平铺不是同一个东西。我现在理解了之前建议我改为“密铺\纹理”的老哥为什么会这样建议了，是他理解错了。诚然翻译为平铺更省事儿，但分块更合适描述这里尝试去做的事情。翻译为平铺确实不知道是在干嘛（比如会有你以及之前那位老哥一样误会了这个功能的用户，所以我更觉得有必要把这两个词区别开来了）
Firstpass height/width - 这个我也很头疼如何翻译，因为他确实要diffuse两遍的。使用第一遍宽度第一遍高度看起来很丑很别扭但能够理解以及足够准确。如果要说，初始高度、初始宽度可能比较好。你觉得呢（这个我先不改了）
Denoising strength - 我不太清楚你想表达什么
Variation - 他这里的做法是**另外添加了一层噪声进去**，而如果使用eular等方法的话，强度为1的时候图的结果是完全等于variation seed的结果。我也考虑过变体、变种、变量和变化等用词。但综合考虑之下（从功能、实现方法、使用目的）我认为差异会让这个部分的意思更容易理解。
Batch count和Batch size - 确实，这样会更方便理解。
Aesthetic text for imgs - 这个我还需要看一下他的具体实现方法。目前改成图集embedding了。
Draw legend - 你试试
resize - /
Poor man's outpainting - 这个我清楚，但不知道怎么意译比较合适。这里的poor是不怎么好的意思。poor man是不太聪明的意思。干脆翻译成效果稍差的outpaint吧
Override - override是无视已有设定覆写为指定值
Randomness - 改为随机度
factor - 改为系数
Pixels to expand - 当前翻译为拓展的像素-建议严谨一些-拓展的像素数
Original prompt - 当前翻译为原始提示词，但一般习惯就和初始值、初始化、初始状态一样，更倾向初始提示词
Scale by和Scale to - 等比缩放、指定尺寸缩放
Images directory - directory 统一为目录，path统一为路径。（虽然实际上没有区别，除了一些模糊的情况下我才会特地指明是文件夹减少理解困难）
number of images to delete consecutively next - 不是机翻。。。不过你的翻译确实更顺口。
favorites - 拿其他软件or公司的翻译视为“标准”是很奇怪的行为。你觉得坐和放宽、比大更大会是标准吗？不过收藏夹确实更合适我的收藏（因为他默认目录里面会有所有你手动点过save的图。。。）
ignore - 忽略确实更合适。
Save a copy of image before doing face restoration - fixed
Unload VAE and CLIP from VRAM when training - 因为我不想要用卸载这个词汇，uninstall在中文也是卸载。but anyway，这个已经是过时的内容了，现在这句的最新版本已经是Move VAE and CLIP那句了。
Filename join string - 改成字符串了，但其实我不太愿意用“字符串”这个词，很不中文（虽然很tech）。但一时半会儿想不到更好的翻译。
When reading generation parameters from text into UI (from PNG info or pasted text), do not change the selected model/checkpoint.  - model和checkpoint是两个不同的东西，但这里就是指同一个东西。
Do not do anything special - 无需做任何事情是非常明显的错误理解。考虑到原翻译确实有点翻译腔了，这里改为：什么都不做

dtlnor committed:
Update localizations/zh-hans.json

Co-authored-by: liggest <43201720+liggest@users.noreply.github.com>

dtlnor committed:
Update localizations/zh-hans.json

Co-authored-by: liggest <43201720+liggest@users.noreply.github.com>

Yorha4D commented:
> 做任何事情是非常明显的错误理解。考虑到原翻译确实有点翻译腔了，这里改为：什么都

@dtlnor 我和@liggest @lockmatrix讨论了下，觉得当前PR活跃度是最高的 应该以此为主pr
#3119 已经喊负责维护的技术关闭那个分支了，现在我想把精力放在这里
话说....来群里划水啊

dtlnor committed:
Update localizations/zh-hans.json

Co-authored-by: liggest <43201720+liggest@users.noreply.github.com>

dtlnor committed:
Update zh-hans.json

- fix according to https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/3348#issuecomment-1287923525

dtlnor commented:
> > 做任何事情是非常明显的错误理解。考虑到原翻译确实有点翻译腔了，这里改为：什么都
> 
> @dtlnor 我和@liggest @lockmatrix讨论了下，觉得当前PR活跃度是最高的 应该以此为主pr #3119 已经喊负责维护的技术关闭那个分支了，现在我想把精力放在这里 话说....来群里划水啊

Thanks for your contribution. 感谢你的理解与贡献，也感谢你对这份翻译的认同。你们对这份翻译作出了非常大的帮助。群我就不加了，如果有什么翻译上的问题我想可以继续在这里讨论。

dtlnor committed:
Update zh-hans.json

byzod commented:
这个tiling和数据分析里的分块完全不是一回事吧，是指图形学里纹理处理相关的一个专用名词，我不知道翻译是啥但反正不是分块

从普通用户的角度来讲看到分块完全不知道在说什么鬼
建议要不参考图形学领域的术语翻译，要不就采用近似含义的平铺/密铺，好歹能猜个五成

或者参考日语翻译直接意译为`纹理生成模式`
https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/774be6d2f271415a82d9a83147e8ee8bbad018d0/localizations/ja_JP.json#L56

dtlnor commented:
> 图形学里纹理处理相关

以esrgan为例，参考[upscale wiki](https://upscale.wiki/wiki/Terminology)的解释：
Tile Size: Most ESRGAN implementations split images into tiles to avoid running out of VRAM. The tile size defines how large these tiles are.
Larger tiles are not automatically better, but they can sometimes avoid seams and slightly speed things up. However, smaller tiles usually work just as well.
又及，https://github.com/xinntao/Real-ESRGAN/issues/197 这个issue里面很明显地提出了基于分块(tiling)实现的放大容易引入的接缝问题。
或者你可以查看我在这里的回答 https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/3119#issuecomment-1287877953
如果你设置sd upscale的分辨率为输入图片分辨率然后设置overlap为0，你可以很清晰地观察到接缝，大概可以理解为什么我会说分块比较合适

batvbs commented:
一个建议：
send to img2img等按钮，与其翻译成发送到/跳转到，不如直接用箭头表示，看起来也更直观。示范：
>>文生图        >> 图生图        >> 修补      >>更多(高清化)

byzod commented:
> > 图形学里纹理处理相关
> 
> 以esrgan为例，参考[upscale wiki](https://upscale.wiki/wiki/Terminology)的解释： Tile Size: Most ESRGAN implementations split images into tiles to avoid running out of VRAM. The tile size defines how large these tiles are. Larger tiles are not automatically better, but they can sometimes avoid seams and slightly speed things up. However, smaller tiles usually work just as well. 又及，[xinntao/Real-ESRGAN#197](https://github.com/xinntao/Real-ESRGAN/issues/197) 这个issue里面很明显地提出了基于分块(tiling)实现的放大容易引入的接缝问题。（虽然、real-esrgan跟esrgan不是一回事） 或者你可以查看我在这里的评论 [#3119 (comment)](https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/3119#issuecomment-1287877953) 如果你设置sd upscale的分辨率为输入图片分辨率然后设置overlap为0，你可以很清晰地观察到接缝，大概可以理解为什么我会说分块比较合适
> 
> 至于你说的数据分析或者图形学纹理处理，恐怕与这里的tiling无关。

好吧，这下就更不懂了（

dtlnor commented:
> 一个建议： send to img2img等按钮，与其翻译成发送到/跳转到，不如直接用箭头表示跳转，看起来也更直观。示范：
> 
> ```
> >>图生图        >>修补        >> 更多(高清化)
> ```
> 
> 图转图看起来怪怪的，文生图，图生图看起来更合理。 "Make Zip when Save?": "保存时生成压缩档?"很明显机翻，`保存时生成zip压缩文件`更合理。

1. 挺好的，直观简洁
2. 从ai意义上来看，确实应该是“生”
3. ~~我看起来真的很像扎克伯格吗~~ 压缩档是我对zip的翻译（我们这边多用档案而不是文件一词），确实应该是压缩文件更适合简体中文的用户习惯。这点是我的疏忽。

dtlnor committed:
Update zh-hans.json

dtlnor commented:
> Tiling

@Yorha4D @byzod 
oh 我终于知道你们在说什么了。非常抱歉，这是绝对是我的问题。
在 Restore faces 旁边那个 Tiling 正正是“可平铺”的意思，他会生成一张上下左右都可以拼接起来的图片。这绝对应该称呼为“平铺/密铺”。我一直以为我们讨论的是 sd upscale / esrgan / SwinIR 里面的tiling。我因为显卡资源足够负荷所以没有实际用过"Tiling"这个功能。直到刚才才发现。两者不是同一个tiling。这是我的失误。非常抱歉。我会改掉他的了

dtlnor committed:
Update zh-hans.json

liggest commented:
怪不得觉得 Tiling 的哪个翻译都有点怪…原来在不同的面板意思不一样

liggest commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,482 @@
+    "Resize image to target resolution. Unless height and width match, you will get incorrect aspect ratio.": "将图像大小调整为目标分辨率。除非高度和宽度匹配，否则你将获得不正确的纵横比",
+    "Resize the image so that entirety of target resolution is filled with the image. Crop parts that stick out.": "调整图像大小，使整个目标分辨率都被图像填充。裁剪多出来的部分",
+    "Resize the image so that entirety of image is inside target resolution. Fill empty space with image's colors.": "调整图像大小，使整个图像在目标分辨率内。用图像的颜色填充空白区域",
+    "How many times to repeat processing an image and using it as input for the next iteration": "重复处理图像多少次并将其用作下一次迭代的输入",
```
```suggestion
    "How many times to repeat processing an image and using it as input for the next iteration": "处理图片并作为下次迭代输入的重复次数",
```

感觉还是有点拗口…

dtlnor replied:
重复处理图像并用作下次迭代输入的次数？

liggest commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,482 @@
+    "Decode CFG scale": "解码提示词相关性(CFG scale)",
+    "Randomness": "随机度",
+    "Sigma adjustment for finding noise for image": "为寻找图中噪点的 Sigma 调整",
+    "Loops": "循环",
```
```suggestion
    "Loops": "迭代次数",
```

liggest commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,482 @@
+    "Click to Upload": "点击上传",
+    "Image": "图像",
+    "Check progress": "查看进度",
+    "Check progress (first)": "查看进度 (1st)",
```
```suggestion
    "Check progress (first)": "(首次)查看进度",
```

不知道这样会不会好一点，虽然根本没找到这个词条是在哪个界面出现的…

dtlnor replied:
https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/6bd6154a92eb05c80d66df661a38f8b70cc13729/modules/ui.py#L562
it's a invisible button...

liggest commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,482 @@
+    "right": "右",
+    "up": "上",
+    "down": "下",
+    "Fall-off exponent (lower=higher detail)": "衰减指数（低=高细节）",
```
```suggestion
    "Fall-off exponent (lower=higher detail)": "衰减指数(越低细节越好)",
```

liggest commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,482 @@
+    "File format for grids": "概览图的文件格式",
+    "Add extended info (seed, prompt) to filename when saving grid": "保存概览时将扩展信息（随机种子、提示词）添加到文件名",
+    "Do not save grids consisting of one picture": "不要保存由一张图像组成的概览图",
+    "Prevent empty spots in grid (when set to autodetect)": "防止概览图中出现空白点（设置为自动检测时）",
```
```suggestion
    "Prevent empty spots in grid (when set to autodetect)": "（在自动检测时）防止概览图中出现空位",
```

虽然没找到哪里可以设置自动检测 \_(:з」∠)\_

liggest commented on localizations/zh-hans.json:
```diff
@@ -0,0 +1,482 @@
+    "When using 'Save' button, only save a single selected image": "使用“保存”按钮时，只保存一个选定的图像",
+    "Do not add watermark to images": "不要给图像加水印",
+    "Paths for saving": "保存路径",
+    "Output directory for images; if empty, defaults to three directories below": "图像的输出目录； 如果为空，则默认为下面三个目录",
```
```json
  "Output directory for images; if empty, defaults to three directories below": "图像的输出目录； 如果为空，则默认为下面三个目录",
  "Output directory for grids; if empty, defaults to two directories below": "概览图的输出目录； 如果为空，则默认为以下两个目录",
```

```suggestion
    "Output directory for images; if empty, defaults to three directories below": "图像的输出目录； 如果为空，则默认为以下三个目录",
```

dtlnor committed:
Update localizations/zh-hans.json

Co-authored-by: liggest <43201720+liggest@users.noreply.github.com>

dtlnor committed:
Update localizations/zh-hans.json

Co-authored-by: liggest <43201720+liggest@users.noreply.github.com>

dtlnor committed:
Update localizations/zh-hans.json

Co-authored-by: liggest <43201720+liggest@users.noreply.github.com>

dtlnor committed:
Update localizations/zh-hans.json

Co-authored-by: liggest <43201720+liggest@users.noreply.github.com>

dtlnor committed:
Update localizations/zh-hans.json

Co-authored-by: liggest <43201720+liggest@users.noreply.github.com>

dtlnor committed:
Update localizations/zh-hans.json

Co-authored-by: liggest <43201720+liggest@users.noreply.github.com>

dtlnor committed:
Update localizations/zh-hans.json

Co-authored-by: liggest <43201720+liggest@users.noreply.github.com>

CCRcmcpe commented:
外补绘制可以参考隔壁，改成外延绘制

batvbs commented:
> 

我觉得绘图、绘制、修补、修改都可以。就不要弄些奇怪的自创词了。

batvbs commented:
> 外补绘制可以参考隔壁，改成外延绘制

我觉得绘图、绘制、修补、修改都可以。就不要弄些奇怪的自创词了。

doubil613 commented:
noise multiplier中的multiplier翻译成乘数/乘子会不会更好一些，倍增器这一称呼在机器学习领域似乎比较少见

liggest commented:
> 外补绘制可以参考隔壁，改成外延绘制

简单粗暴的图像扩展（
虽然这样 paint 的意味就没有了

byzod commented:
原来lz母语是繁中？那为什么卜直接弄个繁中版..

`用在`建议改成`用于`
https://github.com/dtlnor/stable-diffusion-webui/blob/6f95dfb7236fa5656473cf6faf6a91819ff5cee1/localizations/zh-hans.json#L127

所有的`填空`都建议改成`填充`, 包括官方中文windows的壁纸适应模式在内, 我还没有见过哪个软件的fill是译作填空的
https://github.com/dtlnor/stable-diffusion-webui/blob/6f95dfb7236fa5656473cf6faf6a91819ff5cee1/localizations/zh-hans.json#L137

这个选项是指算法2的应用强度吗？还是单纯跑了一边然后按给定透明度和算法1直接叠加？如果是第1种的话是不是意译比较好, 对算法说`可见度`感觉很难理解, 还是说这是业界术语..？
https://github.com/dtlnor/stable-diffusion-webui/blob/6f95dfb7236fa5656473cf6faf6a91819ff5cee1/localizations/zh-hans.json#L200

这里的`prepend`和`append`是指对文件内容的操作, 中文的`放前面`则同时具有`place before`和`prepend`的含义, 建议改为`添加到文件内容前`之类的形式明确含义
https://github.com/dtlnor/stable-diffusion-webui/blob/6f95dfb7236fa5656473cf6faf6a91819ff5cee1/localizations/zh-hans.json#L270

有点拗口, 是不是改成`只有一张图片时不要保存概览图`之类的比较好
https://github.com/dtlnor/stable-diffusion-webui/blob/6f95dfb7236fa5656473cf6faf6a91819ff5cee1/localizations/zh-hans.json#L307

建议txt翻译掉, 文本文件或别的. 其一与其余部分统一, 其二有相当多windows用户是不显示扩展名的, 你说txt他们可能无法理解的, 虽然说能玩ai的多半也不至于txt都不认识..
其余诸如jpg/png/csv如何翻译也需要斟酌
https://github.com/dtlnor/stable-diffusion-webui/blob/6f95dfb7236fa5656473cf6faf6a91819ff5cee1/localizations/zh-hans.json#L267

这个tip是指手输的seed一般都是从哪拷贝的可以方便复制粘贴, 随机种子则只能从log里拷贝, 对不擅长看一大堆参数和英文混杂的log的用户来说"主要是这种情况下有用", 现在的翻译很拗口, 还可能有`如果不是随机出来的话就没用`的歧义, 建议后半改成`...对随机生成的种子很有用`之类的意思
https://github.com/dtlnor/stable-diffusion-webui/blob/6f95dfb7236fa5656473cf6faf6a91819ff5cee1/localizations/zh-hans.json#L438

batvbs commented:
> 原来lz母语是繁中？那为什么卜直接弄个繁中版..
> 
> `用在`建议改成`用于` 
>...

`可见度`不明其意，但由于没有更好的翻译，或许可以保留英语对照 `可见度(visibility)`

`"Existing Caption txt Action"`在web网页里找不到，不明其义，或许你可以解释一下？

关于PNG、jpg、CSV，翻译成PNG图片、jpg图片、CSV表格即可。

dtlnor commented:
> 外补绘制可以参考隔壁，改成外延绘制

其实都有点别扭，没有什么特别恰当的翻译，差别不太大我就保留原来的了

dtlnor commented:
> noise multiplier中的multiplier翻译成乘数/乘子会不会更好一些，倍增器这一称呼在机器学习领域似乎比较少见

~~不如说ML就不应该用中文~~ 
抄一份disco diffusion关于eta的说明：

> [**eta**: (0.5|0-1.0)  eta (greek letter η) is a diffusion model variable that mixes in a random amount of scaled noise into each timestep. 0 is no noise, 1.0 is more noise. As with most DD parameters, you can go below zero for eta, but it may give you unpredictable results.](https://docs.google.com/document/d/1l8s7uS2dGqjztYSjPpzlmXLjl5PM3IGkRWI3IiCuK7g/mobilebasic)

我倾向于使用乘数。噪声乘数。公式参考 https://nn.labml.ai/diffusion/stable_diffusion/sampler/ddim.html

dtlnor commented:
> 原来lz母语是繁中？那为什么卜直接弄个繁中版..
1. fixed
2. 填空的原因参考 https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/3348#issuecomment-1288070570 
3. 不可能是第一种。可以确定的是算法1、2都是独立运行互相不干扰的。具体实现我没看代码，我个人使用的体验就是第二种（很简单调0跟1等价于完全的算法1跟2）。这个功能我也想不到太好的翻译。算法2的覆盖程度？算法二不透明度？（我不认为这是什么专有的terms）
4. 是这样的，这个情况是预处理图集的时候做的事情。如果一个图集预处理之前已经有了txt描述，要么无视掉（然后由deepbooru或者BLIP生成）要么复制过去（不使用deepbooru或者BLIP的时候）要么放前面（原本的描述在前面，deepbooru/BLIP的在后面）要么放后面（反过来）。这个是生成文件中途做的事情（prepend/append the content of the previous text to the newly generated content.）而不是生成了文件之后对新生成的txt以及旧的txt做一个合并操作（虽然结果是一样的）。
5. 脑袋转不过来弯了，感谢
6. 你不要骗我.jpg. 这些都是基础的文件格式，学习成本也不高。能上网冲浪还懂得下webui的不会这个都不懂。
7. 查了查，randomed是固定的意思。改成了`如果想要固定结果就会很有用`

dtlnor commented:
> `"Existing Caption txt Action"`在web网页里找不到，不明其义，或许你可以提供更完美的翻译。
> 
> 关于PNG、jpg、CSV，补充为PNG图片、jpg图片、CSV表格即可。
> 
> 另外，绝大部分简体中文用户不知道`Wiki / 维基`的含义，只知道 百度百科。可以翻译为 帮助 /文献 / 百科 等。

`Existing Caption txt Action` 确实翻得不太好。改了，这个是预处理图集的时候的东西。
github没有中文版。wiki也是指github的wiki。所以就不翻译了。

dtlnor committed:
Update zh-hans.json

dtlnor committed:
re-order some element. update to latest

byzod commented:
> 
>     2. 填空的原因参考 [add support for simplified chinese 简体中文化 #3348 (comment)](https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/3348#issuecomment-1288070570)

我还是坚持填充，首先在填这个动作上，这两个词差不多，但是填空具有非常明确的[填补空缺](https://www.zdic.net/hans/%E5%A1%AB%E7%A9%BA)意象，你也提到了，在生成之前首先会用颜色覆盖原本内容，在会覆盖的前提，我不知道繁中如何，简中语境里是极少用填空的。
其次，正如楼上所说，这个词适用范围实际上基本上限定在考试时的填空题，实际使用时也是用填空题的意象
最后，实际上inpaint在fill时所做的第一步，正是字面意义的填充，之后根据填充颜色生成的功能，则是由sd引入的新的概念，如果没有办法用一个词概括这两个阶段，要么造一个与现有词不冲突的新词，要么使用部分描述其功能的近似名词作为引导，从这个角度来说，从`填充`这个熟悉的动作来延伸，会更容易理解该功能
另外，我真的没骗你，没见过哪个软件用填空的
要么就造新词意译，`填充后脑补`之类的

- ![1](https://user-images.githubusercontent.com/848607/197494569-cb6b939d-3c30-4042-bbcf-698480196a6c.png)
- ![2](https://user-images.githubusercontent.com/848607/197494571-7e40d46f-6ac4-4cd4-b168-c465fce1a8f2.png)
- ![3](https://user-images.githubusercontent.com/848607/197494578-72b94fbb-13b5-46b9-ab7f-e9bad09348eb.png)

dtlnor commented:
> 我还是坚持填充，首先在填这个动作上，这两个词差不多，但是填空具有非常明确的[填补空缺]
~~不 繁中也不怎么用填空~~
其实一开始我想用填白或者填黑的。但他也不是白或者黑所以我才选的填空。也正是取填补空缺的意思，就像是用颜色桶把颜色倒到白色画布一样。通常用fill的话很容易直接把那个被蒙住的element抹掉就没了。而且这四个选项其实都是描述第一阶段的。个人认为第二阶段（指sd生成）不需要囊括进去。
不过比起生僻词新解，近似词延伸确实如你所说更为大家所接纳。我会改为填充的。

nfz commented:
繁中母语来做简中有点奇怪吧，不要再创造新词了。

实在没有翻译的词可以留着英语吧，创造太多新词看不懂的话还不如直接用英语版，哪怕不会英语，大家也快看懂了。

batvbs commented:
我希望尽早合并代码，一些小问题合并后再改。（日文合并后也在改）




"First Page": "第一页",
"End Page": "尾页",
建议此处文字对应，First Page应为“首页”

liggest commented:
话说有考虑过合并后怎么更新的问题吗

dtlnor commented:
> 繁中母语来做简中有点奇怪吧，不要再创造新词了。
> 
> 实在没有翻译的词可以留着英语吧，创造太多新词看不懂的话还不如直接用英语版，哪怕不会英语，大家也快看懂了。

不太合适的偏见。你并不知道我的环境，不建议妄下定论

dtlnor committed:
Update zh-hans.json

dtlnor commented:
> 我希望尽早合并代码，一些小问题合并后再改。目前俄语日语韩语阿拉伯语都合并了，我们还要在这里讨论多久？
> 
> "First Page": "第一页", "End Page": "尾页", 建议此处文字对应，First Page应为“首页”

我也一样，但什么时候合并并不由我决定。或者我应该直接通知他？

dtlnor commented:
@AUTOMATIC1111 We've finished discussion about the translation. Please merge this pr once you have time to review.

nfz commented:
> > 繁中母语来做简中有点奇怪吧，不要再创造新词了。
> > 实在没有翻译的词可以留着英语吧，创造太多新词看不懂的话还不如直接用英语版，哪怕不会英语，大家也快看懂了。
> 
> 不太合适的偏见。你并不知道我的环境，不建议妄下定论
一些上面提的比如 "填充"和"填空"等等 ,中国大陆中文是不会这样用的,但是我发现zh-Hans代表的是简体中文,而不是zh-cn大陆中文,所以你没有完全按大陆的方式翻译也是有道理的,我不再提出异议了.

dtlnor commented:
> > 一些上面提的比如 "填充"和"填空"等等 ,中国大陆中文是不会这样用的,但是我发现zh-Hans代表的是简体中文,而不是zh-cn大陆中文,所以你没有完全按大陆的方式翻译也是有道理的,我不再提出异议了.

As I said before https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/3348#issuecomment-1288899019 繁体也不怎么用。填空只是单纯生僻而已。
简体中文用户最广的群众是中国大陆。所以我个人还是倾向尽可能符合大陆的方式翻译的。我没有试过以这个理由为由反对其他人的修改意见。我做这份翻译很大一部分的理由就是为了服务我熟悉或不熟悉的网友们，而他们都是中国内地的简中用户。

dtlnor committed:
Update zh-hans.json

dtlnor committed:
Update zh-hans.json

dtlnor committed:
Update zh-hans.json

batvbs commented:
windows选择区域为中国时，默认的地区代码是zh_CN
请注意大小写和符号。
![0caa3e9eba05e6723dbfaab2944142dc jpeg (1)](https://user-images.githubusercontent.com/60730393/197765459-b282e8c4-81e0-4b52-96e3-72b8acfa2c9b.png)

dtlnor committed:
change file name

dtlnor commented:
> windows选择区域为中国时，默认的地区代码是zh_CN
> 
> 请注意大小写和符号。
> 
> 
> 
> 不建议使用zh_hans代码，这包括了zh_CN和zh_SG（新加坡）两个分支。
> 
> 类似的，其他语言翻译者均使用本地分支作为文件名称，而不是en（英语）、ar（阿拉伯语）
> 
> 
> 
> 

主要是跟其他的放一块儿统一比较好看。改成zh_CN了

AUTOMATIC1111 commented:
is this ready to be merged in?

dtlnor commented:
> is this ready to be merged in?

Yes, thank you.
